ManualText.txt

```diff
@@ -792,74 +787,57 @@ o	If the driver responds that he or she is just dropping someone off or that the
-•	Drag the Bennett cameras from the system tree into the “drag item here” boxes one by one until the grid is full.
-•	When the grid is full, click the blue arrow icon to “Undock Tab”. This will let the camera grid “float” above the main window.
-•	Once undocked, drag the grid to the right until it is visible on the dual-display monitor to the right. Once on the other monitor, maximize the window to fill the whole screen.
-•	Navigate back to the left hand screen with the main DS ControlPoint window. Select another 3x3 grid for the additional cameras.
-•	Fill the grid with the remaining cameras for Bennett and KOSC. There will be one empty spot left over. We normally put the Physical Plant dumpsters in that box.
-•	Once the grid is full, you do not have to undock it. Simply leave it as it is.
+6.3	Cameras
+Keep an eye on the cameras as much as possible during your shift. This does not mean constantly watching them, but keeping them in your peripheral vision and glancing at them frequently.  If you notice anything suspicious on the cameras during your shift, write down any relevant information on your scratch pad and radio the officer.  It is a good idea to also record the time in case you need to log the information later.
+
+Using the Cameras
+•	There are four camera monitors controlled by two seperate computers. They share a keyboard and mouse, so you must use the black switch-box on the desk to choose which one to operate. 
+•	These computers are designated for camerera monitoring ONLY. They should never be used for any other purpose.
+•	You can make a camera full screen by selecting it from the dropdown menu on the left of the screen.
+•	When a camera is selected in full screen, you can use the controls in the bottom left of the screen to manuever, focus, zoom, etc. 
+o	Note: Only some cameras have the ability to move.
+o	Do NOT unnecessarily move cameras. Some are positioned for specific reasons. If you beleieve a camera should be moved, ask an officer or the ACDC if it should be where it is or if it should be positioned differently.
+o	If you need to operate a camera for an extended period of time, it can be helpful to close the other window on the same computer. This can help to keep the computer from responding slowly while you try to manuever a camera. 
+•	To go back to the standard view, select the appropriately labeled layout (1A, 1B, 2A, or 2B). 
+o	Be aware that there is a camera in our office. We do not monitor it, but you can pull it up from the menu. There is also a camara in KOSC 036 that we do not monitor, but only the officers and the ACDC have access to it. 
+•	Avoid moving cameras if not neccessary. Some will reset themselves to a default angle, but some will NOT. 
```

Comment:
when you do move a camera, unless instructed otherwise return it to it's normal home setting.